server/graphql/v2/object/Host.js

```diff
@@ -1091,23 +1091,36 @@ export const GraphQLHost = new GraphQLObjectType({
 
           const pageQuery = `
                 WITH hostadmins AS (
-                  SELECT "MemberCollectiveId" FROM "Members" WHERE "CollectiveId" = :hostid AND "deletedAt" IS NULL AND role = 'ADMIN'
-                ), orgs AS (
-                  SELECT c.id, c.slug,ARRAY_AGG(DISTINCT m."MemberCollectiveId") as "admins", ARRAY_AGG(DISTINCT t."HostCollectiveId") as hosts
+                  SELECT m."MemberCollectiveId", m."CreatedByUserId" as "UserId"
```

Comment:
`m."CreatedByUserId"` cannot safely be interpreted as "User ID of the member"; it contains the ID of the user who created/invited the member, not the User ID itself. 

I suppose your intention was to write:

```suggestion
                  SELECT m."MemberCollectiveId", u."id" as "UserId"
```

Reply:
Thanks! Nice catch, this was here from the first attempt.